### PR TITLE
Improve wording with better html - Fix tags including HTML

### PR DESCRIPTION
**Pull request by mehdichaouch**
Fix for #170

### CODE REVIEW

File: nab8balld/locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -22,18 +22,18 @@ msgid "Magic 8 Ball"
 msgstr "Gourou"
 
 #: templates/nab8balld/settings.html:10
-msgid "Wake up the guru"
-msgstr "Réveiller le gourou"
```
Comment (nabaztag2018) [inline nab8balld/locale/fr_FR/LC_MESSAGES/django.po:26]: Could you keep the original text ? Thanks!
(Wake up the guru instead of Enable)

**Comment by nabaztag2018** [comment]
I just made a comment regarding keeping the original text. Thanks !

File: nab8balld/locale/fr_FR/LC_MESSAGES/django.po (hunk 1 above)
Comment (f-laurens) [reply nab8balld/locale/fr_FR/LC_MESSAGES/django.po:26]: > Could you keep the original text ? Thanks!

I do have a commit for this (plus other french localization cleanup ).
But how do I push a commit to a PR that is not mine? (never done this before...)

**Comment by f-laurens** [comment]
I have tested this PR, with my two additional commits, and it appears to nicely fix the problems from #170.

File: nab8balld/locale/fr_FR/LC_MESSAGES/django.po (hunk 1 above)
Comment (mehdichaouch) [reply nab8balld/locale/fr_FR/LC_MESSAGES/django.po:26]: ok, I do that.

Comment (f-laurens) [reply nab8balld/locale/fr_FR/LC_MESSAGES/django.po]: Thanks!

Comment (f-laurens) [reply nab8balld/locale/fr_FR/LC_MESSAGES/django.po]: I detected a couple of other glitches in french localisation (missing labels).
Can you please pick up from https://github.com/f-laurens/pynab/tree/pr187 the following additional commit:
https://github.com/f-laurens/pynab/commit/a2800c304f372922fb506e232ad69456863bdb63 and push it to this PR?

**Comment by f-laurens** [comment]
I detected a couple of other glitches in french localisation (missing and residual labels).
Can you please pick up from https://github.com/f-laurens/pynab/tree/pr187 the following additional commits:
https://github.com/f-laurens/pynab/commit/a2800c304f372922fb506e232ad69456863bdb63  and https://github.com/f-laurens/pynab/commit/4315571c7977d65f04ec7db4366f110b3821ad75 and push them to this PR?

**Comment by f-laurens** [comment]
I have tested this PR, inluding my 4 (2+2) additional commits, and it appears to nicely fix the problems from #170.

**Comment by mehdichaouch** [comment]
@f-laurens I take your two commits tomorrow morning.

**Comment by mehdichaouch** [comment]
> Can you please pick up from https://github.com/f-laurens/pynab/tree/pr187 the following additional commits:
> [f-laurens@a2800c3](https://github.com/f-laurens/pynab/commit/a2800c304f372922fb506e232ad69456863bdb63) and [f-laurens@4315571](https://github.com/f-laurens/pynab/commit/4315571c7977d65f04ec7db4366f110b3821ad75) and push them to this PR?

@f-laurens  I take your two commits tomorrow morning.

**Comment by mehdichaouch** [comment]
@nabaztag2018 this PR is ready to merge (thanks @f-laurens).

If they is some new wording to fix, makes a new PR and don't make interdependency between contributors and PR, otherwise PR will become blocked or outdated.

**Comment by f-laurens** [comment]
> I take your two commits tomorrow morning.

Thanks. It looks like the problems from #170 are nicely fixed, together with "clean" french localisation files.
I vote for merging this PR.

**Comment by nabaztag2018** [comment]
In @f-laurens we trust !

**Pull request merged**